guides/CONTRIBUTING.md

```diff
@@ -18,26 +16,158 @@ When filing an issue, make sure to provide:
+
+2. **Reviewing pull requests** &ndash; reviewing Pull Requests is crucial as it may help catch the corner cases or bugs that the developer did not notice. Every review matters as it may help polish quality of the library.
+
+3. **Contributing to Code** &ndash; code-level contributions generally come in the form of pull requests. By contributing to code you help us with solving issues, fixing bugs or introducing new amazing features. If you want to start your adventure with open source it's good idea to take a look at [good first issue](https://github.com/software-mansion/react-native-screens/pulls?q=is%3Apr+is%3Aopen+label%3A%22good+first+issue%22) on github. Read more about [contributing to code](#contributing-to-code). 
+
+### Repository overview 
+
+- `android` &ndash; source code of Android native implementation
+- `common` &ndash; C++ code related to components - shadow nodes and state
+- `cpp` &ndash; C++ code for turbo modules
+- `Example` &ndash; paper version of React Native mobile example app
+- `FabricExample` &ndash; fabric version of React Native mobile example app 
+- `FabricTestExample` &ndash; fabric version of React Native mobile app containing test examples
+- `gesture-handler` &ndash; interop between react-native-screens and react-native-gesture-handler
+- `guides` &ndash; guides for developers
+- `ios` &ndash; source code of iOS native implementation
+- `native-stack` &ndash; native stack v5, this will be deprecated in favor of react-navigation in near feature
+- `react-navigation` &ndash; git submodule containing react-navigation
+- `reanimated` &ndash; interop between react-native-screens and react-native-reanimated
+- `scripts` &ndash; cli utils 
+- `src` &ndash; JS core code 
+- `TestsExample` &ndash; paper version of React Native mobile app containing test examples
+- `TVOSExample` &ndash; React Native for TVOS app wrapper for shared example code
+- `windows` &ndash; react-native-screens support for Windows 
+
+## Handling open issues 
+
+Often understanding and reproducing the problem can be very time consuming task. The great way to help other contributors get up to the speed with solving an issue is providing detailed description and *reproducible* example. The github already has an template for creating issue, nevertheless we still encounter issues that do not have all necessary details, like:
+
+- repository that we can clone and quickly see the problem,
+- very generic reproduction steps,
+- missing description,
+- not related or truncated stack trace. 
+
+What you can do is ask the owner for those details or try provide them by yourself - try to reproduce the problem and provide missing details, so other developer can start debugging straightaway! 🎉
```

Comment:
That was no the point here. My idea is that other contributors can discuss the problem in the comments to fill out missing details, so we have less work with reproduction, not creating another issue.

Reply:
Sure, let's do this then 👍 

```suggestion
What you can do is ask the owner of an issue for such details or try providing them by yourself - try to reproduce the problem and provide missing details, so that other developers could start debugging straightaway! 🎉
```